cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ZookeeperCluster.java

```diff
@@ -225,6 +234,29 @@ public Service generateService() {
         return createService("ClusterIP", ports, getPrometheusAnnotations());
     }
 
+    public NetworkPolicy generateNetworkPolicy() {
+        //TODO
+        ArrayList<NetworkPolicyPeer> from = new ArrayList();
+
+        NetworkPolicyPeer peer = new NetworkPolicyPeer();
+        from.add(peer);
+        ArrayList ports = new ArrayList();
+
+        NetworkPolicyPort port1 = new NetworkPolicyPort();
+        port1.setPort(new IntOrString(2181));
```

Comment:
You should use the `*_PORT` constants rather than int literals.

Aside: I wonder if we can use this `IntOrString` type in our `CpuMemory`, so people don't always have to use a string.